File: test/EntityFramework/VBTests/VbAdvancedPatterns/Partials/BuildingMf.vb
' Copyright (c) Microsoft Open Technologies, Inc. All rights reserved. See License.txt in the project root for license information.
Imports System

Namespace AdvancedPatternsVB

    Partial Public Class BuildingMf

        Public Sub New()

        End Sub

        Public Sub New(ByVal buildingId As Guid, ByVal name As String, ByVal value As Decimal, ByVal address As AddressMf)
            Me.BuildingId = buildingId
            Me.Name = name
            Me.Value = value
            Me.Address = address
        End Sub

    End Class

End Namespace

File: test/EntityFramework/FunctionalTests.Transitional/TestModels/TemplateModels/CsAdvancedPatterns/Partials/BuildingMf.cs
// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved. See License.txt in the project root for license information.

namespace FunctionalTests.ProductivityApi.TemplateModels.CsAdvancedPatterns
{
    using System;

    public partial class BuildingMf
    {
        public BuildingMf(Guid buildingId, string name, decimal value, AddressMf address)
            : this()
        {
            BuildingId = buildingId;
            Name = name;
            Value = value;
            Address = address;
        }
    }
}
